ESP232.ino

```diff
@@ -1,16 +1,27 @@
+#if defined(ESP8266)
+#pragma message "ESP8266 stuff happening!"
+#include <ESP8266WiFi.h>
+#include <ESP8266mDNS.h>
+#define LED_PIN 0
+const char *sync_str = "ESP8266-232-UPD";
```

Comment:
this does not neccessarily have to be named like the target.
we could think of a more generic "UPDATE-CHECK" or such (?)

Reply:
yeah, as written not a must - merely an idea.